js/AndroidDialogPickerNativeComponent.js

```diff
@@ -10,27 +10,63 @@
 
 'use strict';
 
-import {requireNativeComponent} from 'react-native';
+import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
```

Comment:
Adding missing flow types
```suggestion
import * as React from 'react';
import type {Int32} from 'react-native/Libraries/Types/CodegenTypes';
import type {BubblingEventHandler} from 'react-native/Libraries/Types/CodegenTypes';
import type {ColorValue} from 'react-native/Libraries/StyleSheet/StyleSheet';
import type {ViewProps} from 'react-native/Libraries/Components/View/ViewPropTypes';
import type {ComponentType} from 'react';
import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
```